### PR TITLE
Position management #29724117790658

### PR DESCRIPTION
**Pull request by HoangThiTuanDung**

### CODE REVIEW

File: app/Controller/PositionsController.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<?php
+App::uses('Position', 'Model');
+
+class PositionsController extends AppController {
+	var $name = 'Positions';
```
Comment (dinhhoanglong91) [inline app/Controller/PositionsController.php]: Use public, private or protected. Not var.
Btw, what does it variable mean? When is it necessary?

File: app/Controller/PositionsController.php [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+		}
+		$position = $this->Position->findById($id);
+		if ($this->request->is('post')) {
+			$data = $this->request->data;
+			$this->Position->set($data);
+			if ($this->Position->validates()) {
+				$this->Position->id = $id;
+				$newPosition = $data['Position'];
+				if ($this->Position->save($newPosition)) {
+					$this->Session->setFlash(__('Update poisition successful.'));
```
Comment (dinhhoanglong91) [inline app/Controller/PositionsController.php]: poisition -> position

File: app/View/Positions/manage.ctp [hunk 3]
```diff
@@ -0,0 +1,40 @@
+<div class="container">
+	<div class="alert alert-success flash">
+		<?php echo $this->Session->flash(); ?>
+	</div>
+	<?php 
+		echo $this->Form->create('User', ['type' => 'post', 'class' => 'col-md-2 navbar-right']);
+		echo $this->Form->input('search', ['id' => 'search', 'type' => 'text', 'class' => 'form-control', 'placeholder' => __('Search')]);
+		echo $this->Form->end(); 
+	?>
+	<h2>Positions Management</h2> <hr/>
```
Comment (dinhhoanglong91) [inline app/View/Positions/manage.ctp]: Use __()

File: app/Controller/PositionsController.php (hunk 1 above)
Comment (HoangThiTuanDung) [inline app/Controller/PositionsController.php]: The $name attribute should be set to the name of the controller.
http://book.cakephp.org/2.0/en/controllers.html#Controller::$name

**Comment by dinhhoanglong91** [comment]
OK

**Pull request merged**